.github/workflows/dependabot.yaml

```diff
@@ -0,0 +1,92 @@
+# https://docs.github.com/en/actions
+
+name: "Dependabot Merge"
+
+on:
```

Comment:
```suggestion
on: # yamllint disable-line rule:truthy
```

Will fix the error from yamllint.

Reply:
thanks. i see that in the other workflows now.